docs/docs-content/integrations/kubernetes-support.md

```diff
@@ -0,0 +1,72 @@
+---
+
+
+## Overview
+
+We follow a similar lifecycle as established by the Kubernetes community. You can learn more about the official supported Cloud Native Computing Foundation (CNCF) Kubernetes release cycle in the [Kubernetes Releases](https://kubernetes.io/releases/) page. This means we support three minor Kubernetes versions at any given time. We support the current release and the three previous minor version releases, also known as N-3. For example, if the current release is 1.29, we support 1.28, 1.27, and 1.26.
+
+The table below lists the Kubernetes distributions we support and the duration of support.
+
+| Kubernetes Distribution | Supported Minor Versions | Support Duration | Notes |
+| ----------------------- | ------------------ | ---------------- | ----- |
+| CNCF Kubernetes     | N-3                | 14 months        |  The official open-source version of [Kubernetes](https://kubernetes.io/).    |
+| Palette eXtended Kubernetes (PXK) | N-3                | 14 months   | Additional support may be extended. Discuss this with our support team if you need additional support. |
+| Palette eXtended Kubernetes - Edge (PXK-E) | N-3                | 14 months        | Additional support may be extended. Discuss this with our support team if you need additional support. |
+| Other       | N-3                | EOL       | Other distributions available in Palette such as K3s, Microk8s, and RKE2, we only support until their official EOL. The EOL is set by the respective owner. |
+
+
+:::info
+
+Kubernetes follows the [semantic version schema](https://semver.org/). Versions are annotated as x.y.z, where x is the major version, y is the minor version, and z is the patch version.
+:::
+
+We support N-3 Kubernetes minor versions until the official End-Of-Life (EOL). Once we stop supporting the minor version, we initiate the deprecation process. You can learn more about our deprecation process in the [Pack Deprecation](./maintenance-policy.md#pack-deprecations) section. 
+
+
+## Palette eXtended Kubernetes Support
+
+We support CNCF Kubernetes, [Palette eXtended Kubernetes (PXK)](./kubernetes.md) and [PXK-E](./kubernetes-edge.md) for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process. 
+
+The diagram below illustrates the support lifecycle of a Kubernetes version. A deprecated Kubernetes version will no longer receive updates.
+
+![Diagram of the Kubernetes Support Lifecycle. PXK and CNCF is supported for 14 months. After that, normal deprecation flow is initiated.](/integrations_kubernetes-support_support-cycle.png)
+ 
+Additional support may be extended for PXK and PXK-E. Discuss this with our support team if you need extended support. You can contact our support team at [support@spectrocloud.com](mailto:support@spectrocloud.com).
+
+
+This is at spectrocloud support discretion and is not guaranteed.
```

Comment:
```suggestion
This is at Spectro Cloud support discretion and is not guaranteed.
```

Reply:
OMG - this was a test sentence. Thank you so much for flagging it.